tests/framework/db/oci/QueryBuilderUnionTest.php

```
@@ -26,4 +27,38 @@ class QueryBuilderUnionTest extends BaseQueryBuilderUnion
 {
     protected $driverName = 'oci';
     protected static string $driverNameStatic = 'oci';
+
+    /**
+     * Oracle does not support the `RECURSIVE` keyword. Recursion is implicit when a CTE references itself.
+     */
+    public function testBuildWithQueryRecursive(): void
+    {
+        $db = $this->getConnection(true, false);
```

PHPMD (2.15.0)

[warning] Avoid variables with short names like $db. Configured minimum length is 3. (undefined)

(ShortVariable)